### ISSUE TITLE
cli doesn't pass --require args to node properly

### ISSUE THREAD

**Issue by ahdinosaur** [fgnass/node-dev#111]
hey y'all, sweet module. :smile: 

my use case is i want to run

```shell
node-dev -r babel-core/register -r css-modules-require-hook app/server.js
```

but the current way the code is written means at [this point](https://github.com/fgnass/node-dev/blob/master/bin/node-dev#L32), for the above input the `script`, `scriptArgs`, and `nodeArgs` variables are set as:

```js
script = 'babel-core/register'
scriptArgs = [ '-r', 'css-modules-require-hook', 'app/server.js' ]
nodeArgs = [ '-r' ]
```

[getFirstNonOptionArgIndex](https://github.com/fgnass/node-dev/blob/master/bin/node-dev#L6-L11) is returning the index of `babel-core/register`, when it should be still part of the previous `-r`. `node-dev` is expecting args to be passed as `-r=babel-core/register`, but this doesn't work in this case (not sure how options beyond `-r` work).

**Comment by whitecolor** [comment]
+1 needed

**Comment by whitecolor** [comment]
Create PR https://github.com/fgnass/node-dev/pull/114

**Comment by fgnass** [comment]
Landed in v3.0.0.
